plugins/qlcplus.js

```diff
@@ -19,11 +19,14 @@ module.exports.export = function exportQLCplus(library, options) {
     for (const mode of fixture.modes) {
       for (let i=0; i<mode.channels.length; i++) {
         if (mode.channels[i] == null) {
-          mode.channels[i] = 'No Function ' + i;
```

Comment:
Why shouldn't it be possible to reuse the same channel more than once in a mode? But I admit I didn't thought about that case and also didn't test it.